src/style.css

```diff
@@ -0,0 +1,87 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+form {
+  position: relative;
+  margin-top: 20px;
+}
+
+[data-qa="big"] input {
+  box-sizing: border-box;
+  padding-left: 62px;
+  width: 100%;
+  height: 70px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+
+  font-family: Avenir;
```

Comment:
apply font-family to html/body selector and for input use just `font-family: inherit`;
Also, add `fallback` font